runhouse/__init__.py

```diff
@@ -49,12 +49,4 @@
 # Syntactic sugar
 fn = function
 
-# This allows us to natively interact with resources in the object store from a python interpreter on the cluster
```

Comment:
We don't need to initialize this. We can just walk in to a server, and do `obj_store.initialize()` followed by our ops in order to debug